src/libs/actions/IOU.js

```diff
@@ -220,6 +222,7 @@ function requestMoney(report, amount, currency, recipientEmail, participant, com
         chatReportID: chatReport.reportID,
         transactionID: optimisticReportAction.originalMessage.IOUTransactionID,
         reportActionID: optimisticReportAction.reportActionID,
+        createdReportActionID: isNewChat ? optimisticCreatedAction[0].reportActionID : 0,
```

Comment:
To make sure, if the `createdReportActionID ` will be 0 then we will do nothing in Auth? We wont create the CREATED report action? Asking because for other actions we generate new ones if they are zero so we need to make sure this does not happen for CREATED action

Reply:
You're right @mountiny, we're actually creating a CREATED report action every time, even when we don't send a `createdReportActionID`, I just verified this in the code and in the DB, cc @flodnv.

Reply:
Yes, that's the intention